pkg/api/types.go

```diff
@@ -980,7 +980,34 @@ const (
 )
 
 // ServiceStatus represents the current status of a service
-type ServiceStatus struct{}
+type ServiceStatus struct {
+	// LoadBalancer contains the current status of the load-balancer,
+	// if one is present.
+	LoadBalancer LoadBalancerStatus `json:"loadBalancer,omitempty"`
+}
+
+// LoadBalancerStatus represents the status of a load-balancer
+type LoadBalancerStatus struct {
+	// Name is an identifier for the load-balancer, which can be used
```

Comment:
This is intended to tie in to Spec.LoadBalancer as the way by which someone can request a specific pre-existing load balancer.  I'm concerned that the semantics around attaching to an existing load-balancer are somewhat undefined - can I connect you an LB another user owns?  Can I add ports?  Change ports?

Given that we don't really have a good understanding of this facet and that it doesn't currently work on GCE, even!), I propose that we nix this whole feature (requesting a particular LB) for now and re-add it once we have a bit better understanding of how to do it properly (e.g. more like volume claims).  Your opinion: should we remove this status field or just change comments?

@brendandburns @a-robinson This means we would NOT have the ability to request a specific LB IP, which is actually true right now anyway (though you could call it a bug today, after this it would be intended).